rubocop-klaxit/Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
+gem "rubocop-rails", require: false
```

Comment:
Have you tried on a project that is not coded in Rails?
https://github.com/rubocop-hq/rubocop-rails#compatibility

Reply:
Yes, this is not a problem, dependencies are embedded in the gem.